gufe/visualization/mapping_visualization.py

```diff
@@ -130,11 +131,10 @@ def _draw_molecules(
     highlight_color: Tuple[float, float, float, float]
         RGBA tuple for the default highlight color used in the mapping
         visualization
-    atom_mapping: Dict[Tuple[int,int], Dict[int, int]], optional
-        used to align the molecules to each othter for clearer visualization.
-        The structure contains the indices of the molecules in mols as key
-        Tuple[molA, molB] and maps the atom indices via the value Dict[
-        molA_atom_idx, molB_atom_idx]
+    atom_mapping: dict[int, int], optional
```

Comment:
I'm changing the signature of this, since it's an private/internal function.  Previously it would accept arbitrarily many molecules and mapping, but this was untested/unused.  I've narrowed this down to 1 molecule or 2 molecules with a mapping.

Reply:
would we need to turn this back, when going multistate?

Reply:
or would we implement a new function?

Reply:
just a thought, to think about it.